.github/workflows/l10n.yml

```diff
@@ -129,6 +129,176 @@ jobs:
 
         echo "::notice::All Fluent files passed Mozilla Fluent Linter validation"
 
+  l10n_clap_error_localization:
+    name: L10n/Clap Error Localization Test
+    runs-on: ubuntu-latest
+    env:
+      SCCACHE_GHA_ENABLED: "true"
+      RUSTC_WRAPPER: "sccache"
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        persist-credentials: false
+    - uses: dtolnay/rust-toolchain@stable
+    - uses: Swatinem/rust-cache@v2
+    - name: Run sccache-cache
+      uses: mozilla-actions/sccache-action@v0.0.9
+    - name: Install/setup prerequisites
+      shell: bash
+      run: |
+        sudo apt-get -y update ; sudo apt-get -y install libselinux1-dev locales
+        sudo locale-gen --keep-existing fr_FR.UTF-8
+        locale -a | grep -i fr || exit 1
+    - name: Build coreutils with clap localization support
+      shell: bash
+      run: |
+        cargo build --features feat_os_unix --bin coreutils
+    - name: Test English clap error localization
+      shell: bash
+      run: |
+        export LANG=en_US.UTF-8
+        export LC_ALL=en_US.UTF-8
+
+        # Test invalid argument error - should show colored error message
+        echo "Testing invalid argument error..."
+        error_output=$(cargo run --features feat_os_unix --bin coreutils -- cp --invalid-arg 2>&1 || echo "Expected error occurred")
+        echo "Error output: $error_output"
```

Comment:
Hardcoded tests in CI are really painful to run manually...

I'm not fully sure why you can't make a go test out of this? (maybe that doesn't fully test th end-to-end?)

If not possible, I'd move those tests to a shell script, so that we can run them manually more easily.

Reply:
i replicated them in test_sort for quick and easy testing